### PR TITLE
Fix: Removed hardcoded text from XML file

### LINKED ISSUE #1257: Remove Hardcoded text
**Summary:**

There are some text that are hard-coded in the XML files of the project remove them by extracting them as sting resources

### PR DESCRIPTION
**Pull request by rchtgpt**
## Fix Issue:

Fixes #1257 and #1227 

## Description:

I have removed all the existing hardcoded text.

##
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by rchtgpt** [comment]
@luckyman20 please review this

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -354,6 +354,9 @@
     <string name="add_beneficiary_option">Add Third Party Beneficiaries to your account.\nEither
         enter manually or scan account QR code
     </string>
+    <string name="income_generating_loan">Income Generating Loan</string>
+    <string name="not_applicable">Not Applicable</string>
+    <string name="principal_amount_number">143.00</string>
```
Comment (luckyman20) [inline app/src/main/res/values/strings.xml]: Isn't it set through java later? Do we need this?

Comment (rachittt) [reply app/src/main/res/values/strings.xml]: As of now it isn't edited by java.

Comment (rachittt) [reply app/src/main/res/values/strings.xml]: Yes, removed it. Thanks

Comment (rchtgpt) [reply app/src/main/res/values/strings.xml]: Yes, modified it. Thanks!

**Review by luckyman20 — APPROVED** [review]
LGTM

File: app/src/main/res/layout/fragment_review_loan_application.xml [hunk 2]
```diff
@@ -103,7 +104,7 @@
                 android:layout_marginLeft="@dimen/default_padding"
                 android:layout_marginStart="@dimen/default_padding"
                 android:layout_width="match_parent"
-                android:text="143.00"
+                tools:text="@string/dummy_principal_amount"
```
Comment (naman14) [inline app/src/main/res/layout/fragment_review_loan_application.xml:107]: 1 comment, when using tools:text there is no need to use a string resource as its only meant to be used for the layout editor. We can directly put `tools:text="143.00"`

**Pull request merged**